edx_exams/apps/core/signals/handlers.py

```diff
@@ -18,7 +19,7 @@ def listen_for_exam_attempt_submitted(sender, signal, **kwargs):  # pylint: disa
     """
     get_producer().send(
         signal=EXAM_ATTEMPT_SUBMITTED,
-        topic='exam-attempt-submitted',
+        topic=getattr(settings, 'EXAM_ATTEMPT_EVENTS_KAFKA_TOPIC_NAME', ''),
```

Comment:
might make sense to just do this at the top of the file and assign a constant to use for each of these